src/i18n/content/kr/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -373,7 +373,7 @@ mTLS 암호를 생성하려면 [아래 이 섹션](#mtls-how-to-openshift) 의 
 
 <img
   title="new-relic-one-k8s-cluster-explorer-control-plane-parameters.png"
-  alt="New Relic - Kubernetes Cluster Explorer - Control Plane section"
+  alt="New Relic - Kubernetes cluster explorer- Control Plane section"
```

Comment:
Translation file

Reply:
This change doesn't affect the content itself.